lib/util/mime_utils.js

```diff
@@ -166,6 +166,9 @@ shaka.util.MimeUtils = class {
         return 'dtsx'; // DTS:X
       case base === 'mp4a' && profile === 'a9':
         return 'dtsc'; // DTS Digital Surround
+      case base === 'vp9':
```

Comment:
I don't think we need this case, to translate `vp9` to `vp9`. The default case already handles that.

Reply:
Done!